tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -6794,107 +6818,85 @@ TEST_P(OpConverter_FP32_FP16_Test, ConvertSquaredDifference) {
-                          bool align_corners) {
+NodeDef MakeResizeNodeDef(DataType dtype, bool align_corners) {
   Scope s = Scope::NewRootScope();
   auto input = ops::Placeholder(s.WithOpName("input"), dtype);
   auto size = ops::Placeholder(s.WithOpName("size"), DT_INT32);
   auto attrs = typename OpType::Attrs().AlignCorners(align_corners);
-  auto resize = OpType(s.WithOpName(name), input, size, attrs);
+  auto resize = OpType(s.WithOpName("my_resize"), input, size, attrs);
   return resize.operation.node()->def();
 }
 
-template <typename CType>
 struct ResizeTestParams {
   std::vector<int> input_dims;
   std::vector<int> output_resize_dims;
-  std::vector<CType> input_values;
+  std::vector<float> input_value;
   bool align_corners;
   std::vector<int> expected_output_dims;
-  std::vector<CType> expected_nearest_output_values;
-  std::vector<CType> expected_bilinear_output_values;
+  std::vector<float> expected_nearest_output_values;
+  std::vector<float> expected_bilinear_output_values;
 };
 
-template <typename OpType, DataType dtype>
-void TestConvertResize(OpConverterTest* test) {
-  typedef typename EnumToDataType<dtype>::Type CType;
-
-  std::vector<ResizeTestParams<CType>> params{
-      {
-          /*input_dims=*/{1, 2, 1},       // H, W, C
-          /*output_resize_dims=*/{2, 3},  // H_out, W_out
-          /*input_values=*/CastTestVector<float, CType>({2.0f, -1.0f}),
-          /*align_corners=*/false,
-          /*expected_output_dims=*/{2, 3, 1},  // H, W, C
-          /*expected_nearest_output_values=*/
-          CastTestVector<float, CType>({2.0f, 2.0f, -1.0f, 2.0f, 2.0f, -1.0f}),
-          /*expected_bilinear_output_values=*/
-          CastTestVector<float, CType>({2.0f, 0.f, -1.0f, 2.0f, 0.f, -1.0f}),
-      },
-      {
-          /*input_dims=*/{1, 2, 1},       // H, W, C
-          /*output_resize_dims=*/{2, 3},  // H_out, W_out
-          /*input_values=*/CastTestVector<float, CType>({2.0f, -1.0f}),
-          /*align_corners=*/true,
-          /*expected_output_dims=*/{2, 3, 1},  // H, W, C
-          /*expected_nearest_output_values=*/
-          CastTestVector<float, CType>({2.0f, 2.0f, -1.0f, 2.0f, 2.0f, -1.0f}),
-          /*expected_bilinear_output_values=*/
-          CastTestVector<float, CType>({2.0f, 0.5f, -1.0f, 2.0f, 0.5f, -1.0f}),
-      }};
-
+template <typename OpType>
+void TestConvertResize(ParameterizedOpConverterTestBase* test,
+                       ResizeTestParams& p) {
+  test->Reset();
+  // Create resize node.
+  NodeDef node_def = MakeResizeNodeDef<OpType>(test->get_tf_type(),
+                                               p.align_corners);
+
+  std::vector<int> partial_input_shape;
+  if (test->get_trt_mode() == TrtTestMode::kDynamicShape) {
+    partial_input_shape.resize(p.input_dims.size(), -1);
+    // Here all the dimensions are set to a concrete shape. If I comment any of
+    // these lines, then that dimension becames unknown, because it is
+    // initialized to -1.
```

Comment:
These few lines of comment don't look necessary as they are stating the obvious. Let's remove the comment.

Reply:
Fixed